distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text.enso

```diff
@@ -18,6 +19,20 @@ polyglot java import org.enso.base.Text_Utils
      users with efficient concatenation operations.
 @Builtin_Type
 type Text
+    ## ALIAS Count
+       Computes the number of characters in the text.
+
+       ! What is a Character?
+          A character is defined as an Extended Grapheme Cluster, see Unicode
+          Standard Annex 29. This is the smallest unit that still has semantic
+          meaning in most text-processing applications.
+
+       > Example
+          Getting the length of the string "건반(Korean)".
+
+              "건반(Korean)".length
+    length : Integer
+    length self = @Builtin_Method "Text.length"
```

Comment:
Advanturous! Static method is builtin, but instance method not and delegates to the static one. I am suprised it works. But if it does, then OK.

Reply:
I think I don't understand, where is a static method here?